src/components/PayloadDetailsCard.js

```diff
@@ -61,15 +60,14 @@ export default class PayloadDetailsCard extends React.PureComponent {
 				decimals: SUBSTRATE_NETWORK_LIST[SubstrateNetworkKeys.KUSAMA].decimals,
 				unit: SUBSTRATE_NETWORK_LIST[SubstrateNetworkKeys.KUSAMA].unit
 			});
+		} else if (__DEV__) {
```

Comment:
We should not chose those metadata by default. So `isSubstrateDev` should definitely be checked
```suggestion
		} else if (__DEV__ && isSubstrateDev) {
```

Reply:
Addressed in 5c04b93